critiquebrainz/db/review.py

```diff
@@ -575,3 +575,22 @@ def delete(review_id):
         """), {
             "review_id": review_id,
         })
+
+
+def distinct_entities():
+    """Get a list of ID(s) of entities reviewed on CritiqueBrainz.
+
+    Returns:
+        List of dictionaries of ID(s) of entities reviewed.
+        {
+            "entity_id": (uuid),
+        }
+    """
+    with db.engine.connect() as connection:
+        results = connection.execute(sqlalchemy.text("""
+            SELECT
+          DISTINCT entity_id
```

Comment:
I might be a better to write it as `SELECT DISTINCT entity_id` on one line so that it's obvious what's being selected. It got me confused for a moment.

The goal is not to just split up by keywords, but to split up logical parts of a statement so that it's easier to read.